client/dom/table.ts

```diff
@@ -33,7 +33,7 @@ export type Column = {
 	/** left, center, right. If missing it is aligned to the left by default */
 	align?: string
 	/** tooltip describing column content */
-	title?: string
```

Comment:
with this change, can u try to verify if plot code calling renderTable() is not still supplying `.title`?

Reply:
This property was added by me for the profile, I dont think anyone else is using it. Is more clear to name it tooltip.